src/Tools/dotnet-monitor/Commands/GenerateApiKeyCommandHandler.cs

```diff
@@ -116,5 +131,19 @@ private static string GetFormatString(OutputFormat output)
                     throw new InvalidOperationException(string.Format(Strings.ErrorMessage_UnknownFormat, output));
             }
         }
+
+        /// <summary>
+        /// Represents the output format for <see cref="OutputFormat.MachineJson" />.
+        /// </summary>
+        /// <remarks>
+        /// This is intentionally a second copy of the class so that any changes to the first copy will cause a test failure.
+        /// We shouldn't change this format; if you find yourself here editing this, 
+        /// be careful of any downstream dependencies that are depending on this remaining stable.
+        /// </remarks>
+        internal class MachineOutputFormat
```

Comment:
Perhaps clarify **breaking** changes as opposed to any changes.  It should be ok to add properties and most parsers would be resilient to that.